_posts/2022-12-13-dsaawrapped/dsaawrapped.Rmd

```diff
@@ -0,0 +1,106 @@
+---
+title: "DSAA Wrapped"
+description: |
+  Reflecting on 2022.
+author:
+  - name: Chloe Pou-Prom, Meggie Debnath, Derek Beaton
+date: 2022-12-13
+output:
+  distill::distill_article:
+    self_contained: false
+draft: true
+---
+
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = FALSE)
+```
+
+The end of the year is a time for reflection, reminiscing, and sharing screenshots of our Spotify Wrapped. For those who are unfamiliar, Spotify Wrapped is a yearly marketing campaign by Spotify, where they compile streaming data and create a personalized graphic for each user. ^[We surveyed the members on DSAA and found that the most listened music genre is: ???] 
+
+In the spirit of data, end-of-year reflections, and colorful infographics, we thought it would be fun to create our very own DSAA wrapped!
+
+![](dsaa_wrapped.PNG)
+
+Enjoy some stats!!
+
+# Projects
+
+2022 was a busy year with lots of projects!!
+
+- Total \# of projects deployed: 7
+
+- Most viewed project: Ops Centre
+
+    - The Ops Centre project is....
+    
+- Project with the most team members whose names start with _M_^: Hemodialysis
+
+    - The Hemodialysis project is...
+    
+
+# Product families
+
+2022 was the year of _product families_.
+
+
+![](https://slack-imgs.com/?c=1&o1=ro&url=https%3A%2F%2Fimages.squarespace-cdn.com%2Fcontent%2Fv1%2F58b3cae5d1758e0132a008c6%2F1582078446179-VU0VAYNPOV4P3JIBKNIY%2FIt%2527s%2BAbout%2BFamily%2BLogo%2B1500.png%3Fformat%3D1000w)
+
+What's a product family?
+
+> A product family is....
+
+
+Some product family stats!
+
+- Number of product families: ??
+
+- Product family with the most projects: Patient Flow??
```

Comment:
did a rough count of the sub-folders in each of the PF clickup spaces - this is what i found based on old/new projects combined!
- patient flow: 8
- ai triage: 10
- medical imaging: 2
- workforce optimization: 11
- decision guidance: 9
- data fulfillment: 3